api/src/main/java/org/openmrs/api/db/ProgramWorkflowDAO.java

```diff
@@ -277,4 +280,19 @@
 	 * @return - A List of ProgramWorkflowStates
 	 */
 	public List<ProgramWorkflowState> getProgramWorkflowStatesByConcept(Concept concept);
```

Comment:
@brayoh Add public access modifier to each method in this class

Reply:
I've added a public modifiers for all the methods